tests/api/test_api.py

```diff
@@ -20,13 +20,11 @@ def test_create_api(cript_api: cript.API) -> None:
     """
     tests that an API object can be successfully created with host and token
     """
-    # api = cript.API(host=None, api_token=None)
-    #
-    # # assertions
-    # assert api is not None
-    # assert isinstance(api, cript.API)
+    api = cript.API(host=None, api_token=None)
```

Comment:
This was commented because we use a fixture for this now.
We can have it, it doesn't hurt (as long as you can connect twice to cript).

But I would also be OK with deleting it entirely.

Reply:
sure,